.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+Thanks! for testing out CircuitPython. Now that you have got a problem... you can
+file a bug report for it. Feel free to modify the below format to better suit your issue.
+Also, don't forget to remove me... :)
```

Comment:
```suggestion
<!-- Thanks! for testing out CircuitPython. Now that you have got a problem... you can
file a bug report for it. Feel free to modify the below format to better suit your issue. -->
```

HTML comments work in GitHub Markdown. Comments are usually used for instructions like this.